bescort.lic

```diff
@@ -610,8 +610,58 @@ class Bescort
     end
     case mode
     when /oasis/
-      echo 'Not implemented'
-      exit
+    loop do
+      desert_enter
+      path = %w[s ne ne s s ne e]
+      path.each do |movement|
```

Comment:
What do you think about extracting this loop into a function to reduce duplicated code?

If you don't have time, I'm fine with merging this as-is, too.